pkgs/applications/science/math/mathmod/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, qmake
+}:
+
+mkDerivation {
+  pname = "mathmod";
+  version = "unstable-2023-09-23";
+
+  src = fetchFromGitHub {
+    owner = "parisolab";
+    repo = "mathmod";
+    rev = "efcf329723cb1c5b836e1dd7edb76d67e81f5dbb";
+    hash = "sha256-T6MO+aV3FejlUAgCEr1yJsiwQPZkmc6YRMcPtBq2jF4=";
+  };
+
+  patches = [ ./fix-paths.patch ];
+
+  postPatch = ''
+    substituteInPlace MathMod.pro --subst-var out
+  '';
+
+  nativeBuildInputs = [ qmake ];
+
+  meta = {
+    description = "A mathematical modelling software";
+    homepage = "https://github.com/parisolab/mathmod";
+    license = lib.licenses.gpl2Only;
```

```suggestion
    license = lib.licenses.gpl2Plus;
```
See e.g. https://github.com/parisolab/mathmod/blob/efcf329723cb1c5b836e1dd7edb76d67e81f5dbb/main.cpp#L5-L8

---

Nice catch, thanks